module/applications/compendium-browser.mjs

```diff
@@ -506,7 +916,7 @@ export default class CompendiumBrowser extends HandlebarsApplicationMixin(Applic
     }
 
     else {
-      for ( const child of target.closest(".type-group").querySelectorAll('input[type="checkbox"][value]') ) {
+      for ( const child of target.closest(".type-group").querySelectorAll("dnd5e-checkbox[value]") ) {
```

Comment:
Small fix to prevent the checkbox from getting stuck in an indeterminate state when you click on it.

```suggestion
      target.indeterminate = false;
      for ( const child of target.closest(".type-group").querySelectorAll("dnd5e-checkbox[value]") ) {
```